app/main.py

```diff
@@ -189,3 +190,34 @@ async def get_version_at_date(
             f"Issue parsing date : {str(exception)}. Expected format is YYYY-MM-DD.",
         ) from exception
     return finder.get_version_at_date(parsed_date)
+
+
+@app.get(f"{BASE_PATH}/graph_services/v1/")
+@limiter.limit(RATE_LIMIT)
+async def graph_services(request: Request):
+    """
+    Returns a JSON object with monthly statistics about tracked services.
+    """
+    dataset = CGUsDataset(Path(CGUS_DATASET_PATH))
+    stats = dataset.get_stats()
+    data = pd.DataFrame.from_dict(stats, orient="index")
+    data.sort_values(by="date", inplace=True)
+    data["year_month"] = data.date.dt.to_period("M")
+    over_years = data.groupby("year_month", as_index=False).agg(
+        services_tracked=("service", "unique"),
+        n_services_tracked=("service", pd.Series.nunique),
```

Comment:
I would be in favor of having on homogeneous naming

n_services_tracked
n_services_unique

Also I find n_services_tracked misleading as we are indeed tracking all services but only few are active

So I would go for
- n_services_tracked -> n_services_active
- n_unique_services -> n_services_tracked

What do you think?

Reply:
done